packages/vaadin-grid-pro/test/grid-pro.test.js

```diff
@@ -2,6 +2,7 @@ import { expect } from '@esm-bundle/chai';
 import sinon from 'sinon';
 import { GridElement } from '@vaadin/vaadin-grid/src/vaadin-grid.js';
 import { GridColumnElement } from '@vaadin/vaadin-grid/src/vaadin-grid-column.js';
+import '@vaadin/vaadin-template-renderer';
```

Comment:
I forgot to import the template renderer while removing the Template API as all the test cases in this file successfully pass regardless of [a provided template](https://github.com/vaadin/web-components/pull/2101/commits/6ac141e3500bc0a1613fafa8a1c35904bd579633#diff-1cc413c1143cd9d279a4660317e5d3929e3a7974eddbc141f3f82960050f71a0R16-R18) (as far as tests don't actually use this template).

This missed import has become noticeable since we show the warning:

![image](https://user-images.githubusercontent.com/5039436/121668980-876e2a00-cab4-11eb-82c6-0937e57e6aea.png)